src/components/Profile/Profile.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from 'react';
+import firebase, { getCurrentUser } from '../../services/firebase';
+
+export default function Profile() {
```

Comment:
This implementation may change a decent amount once we are keeping track of the User at all times. But this is definitely good for now. Just a heads up.